payments_py/api/base_payments.py

```diff
@@ -19,33 +19,45 @@ class BasePaymentsAPI:
     It provides common functionality such as parsing the NVM API Key and getting the account address.
     """
 
-    def __init__(self, options: PaymentOptions):
+    def __init__(self, options: Union[PaymentOptions, Dict[str, Any]]):
```

Comment:
Is there a reason to change the way this is initialized. It seems that having only one way is the simplest